backup/setup/migrations.json

```diff
@@ -86,5 +86,16 @@
 			"$date": "2022-11-21T15:44:50.170Z"
 		},
 		"__v": 0
+	},
+	{
+		"_id": {
+			"$oid": "638622fdc5fa134eb24df95f"
+		},
+		"state": "up",
+		"name": "add-submission-keys",
```

Comment:
Maybe more descriptive: "add-submitted-and-graded-keys-to-submission"

Reply:
Done